yrs/src/update.rs

```diff
@@ -697,6 +834,117 @@ impl<T: Iterator> Memoizable for T {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub(crate) enum BlockCarrier {
+    Block(Block),
+    Skip(BlockRange),
```

Comment:
`BlockCarrier` is essentially a wrapper around `Block` which extends it to cover `Skip` semantics for update structure purposes.